packages/components/src/components/carousel/carousel-styles.ts

```diff
@@ -5,28 +5,39 @@ import {
   getBackfaceVisibilityJssStyle,
   getScreenReaderOnlyJssStyle,
   getThemedColors,
-  pxToRemWithUnit,
 } from '../../styles';
 import {
   gridWidthMax,
   gridSafeZone,
-  headingMediumStyle,
   textSmallStyle,
   getMediaQueryMin,
-  gridGap,
+  borderRadiusSmall,
+  headingXLargeStyle,
+  spacingFluidMedium,
+  spacingStaticXSmall,
+  spacingStaticSmall,
+  spacingFluidXSmall,
+  spacingFluidLarge,
 } from '@porsche-design-system/utilities-v2';
+import type { HeaderAlign } from '../../types';
 
 export const bulletActiveClass = 'bullet--active';
 
 const mediaQueryS = getMediaQueryMin('s');
 const mediaQueryXl = getMediaQueryMin('xl');
+// the speed which "splide" framework uses to switch between slides
+const splideSpeed = '0.4s';
```

Comment:
Kind of risky to redefine this value. Safer would be to get it from splide to be in sync.

Reply:
@denyo it's not possible to get splide options on this point of time, because `attachComponentCss` happens during Stencil's `render`, which is being called before Stencil's `componentDidLoad` (where splide is initialized). But what I can do is to define splide's speed explicitly, which will provide also more clarity.

Reply:
Done